ste/sender-blockBlobFromURL.go

```diff
@@ -64,10 +62,6 @@ func newURLToBlockBlobCopier(jptm IJobPartTransferMgr, destination string, pacer
 
 // Returns a chunk-func for blob copies
 func (c *urlToBlockBlobCopier) GenerateCopyFunc(id common.ChunkID, blockIndex int32, adjustedChunkSize int64, chunkIsWholeFile bool) chunkFunc {
-	if blockIndex == 0 && adjustedChunkSize == 0 {
```

Comment:
We should also add a test case that matches up with what the customer was trying to do

Reply:
Customer corrupted the md5 sum. This is another content validation test and is taken care of.